server/tserver/src/main/java/org/apache/accumulo/tserver/TabletServer.java

```diff
@@ -358,6 +359,53 @@ public TabletServer(ServerConfigurationFactory confFactory, VolumeManager fs) th
     this.logSorter = new LogSorter(instance, fs, aconf);
     this.replWorker = new ReplicationWorker(this, fs);
     this.statsKeeper = new TabletStatsKeeper();
+    final int numTopTabletsToLog = aconf.getCount(Property.TSERV_LOG_TOP_TABLETS_COUNT);
+    final long logTopTabletsDelay = aconf.getTimeInMillis(Property.TSERV_LOG_TOP_TABLETS_INTERVAL);
+    if (numTopTabletsToLog > 0) {
+      SimpleTimer.getInstance(aconf).schedule(new Runnable() {
```

Comment:
Would be nice to have some comments explaining this thread.

Reply:
will add